CHANGELOG.md

```diff
@@ -27,6 +27,9 @@
 - Fixed a bug where incorrect syntax error message were shown,
   when using `:` or `=` in wrong possitions in expressions.
 
+- Improved an error message when using discarded names in constructs like:
+  `let some = _func()` or `case { 1 -> _func() }`
```

Comment:
"discarded name" doesn't mean anything outside the compile codebase, so let's call it a variable name starting with an underscore.

Could you add your name here please 🙏

Reply:
Done!